### DISCUSSION TITLE
After install unleash java SDK 8.2.0, I got error every time that call API `gson.fromJson<Map<String…rString, Map::class.java) must not be null`

### DISCUSSION THREAD

**Discussion by nakharin** [Unleash/unleash#4251] [Q&A]
I got error `gson.fromJson<Map<String…rString, Map::class.java) must not be null` every time when i call API after install `implementation("io.getunleash:unleash-client-java:8.2.0")` into BFF mobile using with Kotlin, Ktor CIO

**Comment by sighphyre** [comment]
Hey @nakharin please can you open an issue on the repository here: https://github.com/Unleash/unleash-client-java/issues

If you can include as much detail as you can and if possible the full stack trace that would help us look into this faster
